jest.config.js

```diff
@@ -6,9 +8,10 @@ module.exports = {
     'ts-jest': {
       tsconfig: '<rootDir>/tsconfig.spec.json',
       stringifyContentPathRegex: '\\.html$',
-      astTransformers: {
-        before: ['jest-preset-angular/build/InlineFilesTransformer', 'jest-preset-angular/build/StripStylesTransformer']
-      }
+      astTransformers: [
+        'jest-preset-angular/build/InlineFilesTransformer',
```

Comment:
This one is opposite of what you have done for debug config. Why so?

Reply:
I was doing some tests because it wasn't working but I already fixed this part.

Reply:
okay